Nil pointer exception when ~/.civo.json does not exist

I installed `civo` at 7dc15bb.

When I run the command:

```sh
% civo --region NYC1 kubernetes list
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0xa5fac9]

goroutine 1 [running]:
github.com/civo/cli/cmd.glob..func48(0x1391520, 0xc0004e8990, 0x1, 0x3)
        /home/mvalais/go/pkg/mod/github.com/civo/cli@v0.7.13/cmd/kubernetes_config.go:29 +0x69
github.com/spf13/cobra.(*Command).execute(0x1391520, 0xc0008ae980, 0x3, 0x4, 0x1391520, 0xc0008ae980)
        /home/mvalais/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:854 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x1385040, 0xc000000180, 0x200000003, 0xc000000180)
        /home/mvalais/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/mvalais/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/civo/cli/cmd.Execute()
        /home/mvalais/go/pkg/mod/github.com/civo/cli@v0.7.13/cmd/root.go:26 +0x31
main.main()
        /home/mvalais/go/pkg/mod/github.com/civo/cli@v0.7.13/main.go:23 +0x25
```

I don't get a nil pointer exception when `--region NYC1` is not given:

```sh
% civo kubernetes list
Error: Creating the connection to Civo's API failed with NoAPIKeySuppliedError: no API Key supplied, this is required
```

Would it be possible to add a message that suggests to log into Civo, i.e.,
1. open https://www.civo.com/account/security
2. copy the API key,
3. run `civo apikey add boring_wozniak API_KEY`.

--- a/cmd/kubernetes_list.go
+++ b/cmd/kubernetes_list.go
@@ -29,12 +29,13 @@ If you wish to use a custom format, the available fields are:
 
 		client, err := config.CivoAPIClient()
 
-		if regionSet != "" {
-			client.Region = regionSet
-		}
 		if err != nil {
 			utility.Error("Creating the connection to Civo's API failed with %s", err)
 			os.Exit(1)
+		}
+
+		if regionSet != "" {
+			client.Region = regionSet
 		}
 
 		kubernetesClusters, err := client.ListKubernetesClusters()

